2_process.smk

```diff
@@ -60,13 +60,101 @@ rule interpolate_mntoha_obs_depths:
         "2_process/src/make_obs_interpolated.py"
 
 
+# Convert 7b_temp_merge/out/merged_temp_data_daily.feather to csv
+rule convert_model_prep_obs_to_csv:
+    input:
+        in_file="2_process/in/model_prep/temperature_observations/merged_temp_data_daily.feather"
+    output:
+        csv_file="2_process/tmp/model_prep/temperature_observations.csv"
+    script:
+        "2_process/src/convert_feather_to_csv.py"
+
+
+# Add column of observation depths interpolated to nearest modeling mesh node
+rule interpolate_model_prep_obs_depths:
+    input:
+        "2_process/tmp/model_prep/temperature_observations.csv"
+    output:
+        "2_process/tmp/model_prep/temperature_observations_interpolated.csv"
+    params:
+        depths=config["depths"]
+    script:
+        "2_process/src/make_obs_interpolated.py"
+
+
 # Add elevation to MNTOHA lake metadata
```

Comment:
Suggested edit: To help keep the snakemake file easy to read now that there are 2 datasets and so many rules and functions, you  might consider splitting the rules into two sections `##### Prep MNTOHA data #####` and `##### Prep model-prep data #####` and/or adding section headers, e.g., `##### Interpolate observed temperatures to specified depths #####`, `##### Prepare lake metadata #####`, `##### Prepare sequences for training and testing #####`

Reply:
I like the idea of grouping MNTOHA rules and model-prep rules, but I think some later rules in `2_process.smk` will be shared by both datasets. The next PR will complete this snakefile. Once that PR is complete, I think it will be easier to see which form of organization makes the most sense. I'll implement these section headers then.